modules/@apostrophecms/doc-type/index.js

```diff
@@ -659,7 +664,10 @@ module.exports = {
         }, {
           $set: {
             modified: false,
-            lastPublishedAt: new Date()
+            lastPublishedAt
```

Comment:
What's the difference between `updatedAt` on a published document and `lastPublishedAt`?

Reply:
While we're not doing it at the moment, I foresee scenarios where direct writes to the published locale could happen, for instance to update programmatically managed data that isn't subject to draft/published workflow. This would update `updatedAt` and I want to make sure we have a separate and accurate `lastPublishedAt`.